staging/src/k8s.io/cloud-provider/controllers/node/node_controller.go

```diff
@@ -620,7 +620,7 @@ func (cnc *CloudNodeController) getInstanceNodeAddresses(ctx context.Context, no
 	}, nil
 }
 
-func getCloudTaint(taints []v1.Taint) *v1.Taint {
+func GetCloudTaint(taints []v1.Taint) *v1.Taint {
```

Comment:
I would leave this function unexported and just copy it locally from staging/src/k8s.io/legacy-cloud-providers/azure/azure.go

Reply:
Done!